### ISSUE TITLE
[Accessibility]: Upload/Index timeline time between states is not explained as such

### ISSUE THREAD

**Issue by Strum355** [sourcegraph/sourcegraph#35262]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/33513, #33512

### Problem description

The timeline view lists the amount of time between each state "about 5 hours" etc, but it is not clear via screen reader (and also visually) what this time duration represents.

![image](https://user-images.githubusercontent.com/18282288/167705517-2e230b0e-098f-4e82-b1c5-ab32fdfa8779.png)

### Expected behavior

Reword so the meaning of the time duration is clear both visually and from the screen reader.

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by umpox** [comment]
@Strum355 Can you give an example of this change? 

E.g. Changing `about 5 hours` to `about 5 hours, then` ?

**Comment by Strum355** [comment]
Something like `Took about 5 hours`/`Step took about 5 hours` etc etc @umpox

**Comment by umpox** [comment]
Thanks, @gitstart-sourcegraph please take a look!

**Comment by umpox** [comment]
Fixed in the above PR!
